src/main/kotlin/no/nav/familie/pdf/pdf/domain/PdfStandard.kt

```diff
@@ -1,4 +1,4 @@
-package no.nav.familie.pdf.pdf.types
+package no.nav.familie.pdf.pdf.domain
 
 data class Standard(
     val samsvarer: Boolean,
```

Comment:
Denne koden kan legges i samme fil som `PdfMedStandarder`.

Tror også dere burde fjerne fnuttene som ligger rundt `1a` og nedover til `4e`.

Jeg synes også at `Standarder`-objektet blir litt vel statisk og stort slik det er nå. Jeg foreslår derfor at dere endrer på `PdfMedStandarder` til følgende:
`data class PdfMedStandarder(
    val pdf: ByteArray,
    val standarder: List<Standard>,
)`

Deretter så legger dere til en enum for pdf-kriteriene:
`enum class PdfKriterium{
UA_1,
UA_2,
1_A,
....
} `

Deretter endrer dere på `Standard` til følgende:
`data class PdfStandard(
   val kriterium: PdfKriterium;
   val erOppfylt: boolean;
   val beskrivelse: String;
`

Slik blir PdfMedStandarder-objektet mer dynamisk. Det blir også lettere for frontend å forholde seg til en liste av standarder når den skal mappe ut og vise de oppfylte/ikke-oppfylte standardene med begrunnelser